libs/shared/ui/src/lib/components/accordion/accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import * as AccordionComponent from '@radix-ui/react-accordion'
+import Icon from '../icon/icon'
+
+export interface AccordionProps {
+  header: React.ReactNode
+  children: React.ReactNode
+  open?: boolean
+  disabled?: boolean
+  className?: string
+}
+
+export function Accordion(props: AccordionProps) {
+  const { header, children, open = false, disabled = false, className = '' } = props
+
+  return (
+    <AccordionComponent.Root
+      type="single"
+      defaultValue={open ? 'accordion' : undefined}
+      disabled={disabled}
+      collapsible
+      className={className}
+    >
+      <AccordionComponent.Item value="accordion" className="accordion rounded border border-element-light-lighter-400">
+        <AccordionComponent.Header className="accordion__header h-12">
+          <AccordionComponent.Trigger className="w-full h-full flex justify-between items-center px-4 gap-4">
+            {header}
+            <Icon name="icon-solid-caret-down" className="accordion__arrow text-sm text-text-400 shrink-0" />
+          </AccordionComponent.Trigger>
+        </AccordionComponent.Header>
+        <AccordionComponent.Content className="accordion__content !block border-t border-element-light-lighter-400">
```

Comment:
Why you didn't add `!block border-t border-element-light-lighter-400` on your `accordion__content` class, and it's very useful to have a class for this?

Same remark for all your classes: `accordion__header`

Reply:
Yes, I can add this on the css directly ! 
For the class, I use it for the css animations